content/livesync/models/models.textile

```diff
@@ -0,0 +1,295 @@
+---
+title: Models
+meta_description: "A model is a live, observable data model backed by your database that you can use to keep your frontend applications updated in realtime with the latest information from your database."
+product: livesync
+languages:
+  - javascript
+---
+
+A model is a single instance of a live, observable data model backed by your database. A model requires a "sync function":#sync to tell the model how to initialize with the latest data, and a "merge function":#merge to calculate the next version of the model state when change events are received from the backend database.
+
+You can then "subscribe":#subscribe to a model to receive updates in realtime whenever the model state changes.
+
+<aside data-type='note'>
+<p>You can access the Models SDK directly by referring to ABly's "API references":https://sdk.ably.com/builds/ably-labs/models/main/typedoc/.</p>
```

Comment:
Capital `AB` on Ably.

I'd also suggest rewording this to something more simple, e.g: "View the API reference for the Models SDK".

Reply:
done!

Reply:
2ed8a89002398d33e43e2be6b052818b7237967d